docker/README.md

```diff
@@ -24,6 +24,24 @@ docker run -d -p 8000:8000 remotestorage/armadietto:latest
 
 The armadietto will run as the armadietto user (UID 6582) in the container.
 
+### Behind a Proxy
```

Comment:
It might be best to move this to the notes directory, beside the Apache notes. Reverse proxy config should be the same whether Armadietto is run using Docker or node.

Reply:
Ah yeah, could do. I was thinking this file could be the readme that is on the docker hub page for the image (eg https://hub.docker.com/repository/docker/bytesnz/armadietto), so was putting all information to get up and running. Could add a see more link to the notes folder?

Reply:
> Could add a see more link to the notes folder?

Good idea.

Reply:
I've updated the README ([cefdcb4](https://github.com/remotestorage/armadietto/pull/37/commits/cefdcb4787508f9d897fc1c6a99e29d56c0b995e)), cleaning the organisation up a little bit and added some references to the docs in the git repo.